spaces/S000192/properties/P000027.md

```diff
@@ -0,0 +1,9 @@
+---
+space: S000192
+property: P000027
+value: true
+---
+
+Witnessed by the basis of sets including
```

Comment:
sets --> open sets
including --> consisting of (otherwise seems it includes those, but could include more)